docs/docs/models-definition.md

```diff
@@ -1,25 +1,30 @@
 ## Definition
 
-To define mappings between a model and a table, use the `define` method. Sequelize will then automatically add the attributes `createdAt` and `updatedAt` to it. So you will be able to know when the database entry went into the db and when it was updated the last time. If you do not want timestamps on your models, only want some timestamps, or you are working with an existing database where the columns are named something else, jump straight on to [configuration ][0]to see how to do that.
+To define mappings between a model and a table in v4, you define a class that inherits from [`Sequelize.Model`](../api/model) and call [`Model.init(attributes, options)`](../api/model#init). The model name is determined by the class name. You need to pass the sequelize instance as `options.sequelize` (see example)
+Sequelize will then automatically add the attributes `createdAt` and `updatedAt` to it. So you will be able to know when the database entry went into the db and when it was updated the last time. If you do not want timestamps on your models, only want some timestamps, or you are working with an existing database where the columns are named something else, jump straight on to [configuration ][0]to see how to do that.
 
+Defining a model with [`sequelize.define()`](../api/sequelize#define) as in previous versions is still supported. Check the v3 docs for examples.
 
 ```js
-var Project = sequelize.define('project', {
+class Project extends Sequelize.Model { }
+Project.init({
   title: Sequelize.STRING,
   description: Sequelize.TEXT
-})
+}, { sequelize })
```

Comment:
> I'm not a fan of your ES7 example, why not use statics rather than decorators?

Because the decorators can normalize their input before decorating the class, while you can't if you are declaring statically for the reasons already mentioned. I admit though that from my 3 example it is definitely the most ugly one. If I wasn't using TS I would just go with `init()`.

> I think there's a good case for having options be explicit rather than an object (especially for stuff like name, schema and table).

Yes, I agree, but that's not how it is implemented right now. I just started from how it is atm because it is less code to change.

> Having attributes as an overall decorator also removes the options of having attribute level decorators in the future

That is not true - see my TypeScript example. We can supply both. Actually I am working on a decorator library I want to publish on NPM to make the first two examples possible with 4.0.0-1 and experiment with it in the projects I am involved in.